### PR TITLE
updates to use new input for single value CV

### PR DESCRIPTION
**Pull request by straleyb**

### CODE REVIEW

**Comment by straleyb** [comment]
gotta fix some view stuff, add tests for the new input and update the other single value CVs

**Comment by straleyb** [comment]
Screen Shot 2019-06-06 at 3 03 21 PM

File: app/views/records/edit_fields/_workType.html.erb [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <%= f.input key,
-            as: :controlled_vocabulary,
+            as: :single_value_controlled_vocabulary,
```
Comment (CGillen) [inline app/views/records/edit_fields/_workType.html.erb]: `:workType` is obligation {0, n} on the MAP.
Try `:license`, `:access_restrictions`, or `:repository`

Comment (CGillen) [reply app/views/records/edit_fields/_workType.html.erb]: Or are you thinking of `:resource_type`?

**Comment by straleyb** [comment]
Im pretty sure it was workType. @wickr ?

File: app/indexers/generic_indexer.rb [hunk 2]
```diff
@@ -10,12 +10,14 @@ def generate_solr_document
       rights_statement_labels = OregonDigital::RightsStatementService.new.all_labels(object.rights_statement)
       license_labels = OregonDigital::LicenseService.new.all_labels(object.license)
       language_labels = OregonDigital::LanguageService.new.all_labels(object.language)
-      type_labels = OregonDigital::TypeService.new.all_labels(object.resource_type)
+      type_label = OregonDigital::TypeService.new.all_labels(object.resource_type)
```
Comment (CGillen) [inline app/indexers/generic_indexer.rb]: Changes to this file doesn't seem necessary. Maybe the plurality.

File: .rubocop_fixme.yml [hunk 3]
```diff
@@ -38,6 +38,7 @@ Metrics/MethodLength:
     - 'lib/oregon_digital/triple_powered_properties/inputs/triple_powered_property_input.rb'
     - 'lib/oregon_digital/triple_powered_properties/work_behavior.rb'
     - 'lib/tasks/oregon_digital/csv_ingest.rake'
+    - 'app/indexers/generic_indexer.rb'
```
Comment (CGillen) [inline .rubocop_fixme.yml]: Fixed by removing changes in `app/indexers/generic_indexer.rb`

**Pull request merged**